builder/commands/update.py

```diff
@@ -38,17 +40,19 @@ def take_action(self, parsed_args):
             print parser.print_help()
             sys.exit(1)
         config = self.parse_config(parsed_args.conf)
-        yaml_file = os.path.join(parsed_args.yaml)
-
-        if os.path.isdir(yaml_file):
-            views = [view for view in os.listdir(yaml_file)]
-            for view_yaml in views:
-                self.log.debug("View file %s" % view_yaml)
-                self.read_update(config,
-                                 os.path.join(yaml_file + '/%s' % view_yaml))
-        else:
-            self.log.debug("View file %s" % yaml_file)
-            self.read_update(config, os.path.join(yaml_file))
+        for yaml_filename in parsed_args.yaml:
+
+            yaml_file = os.path.join(yaml_filename)
+
+            if os.path.isdir(yaml_file):
+                views = [view for view in os.listdir(yaml_file)]
```

Comment:
I somehow overlooked this section, wondering why it was not the same way at the test action.

I agree that now it's a bit confusing, maybe I can roll-back my extension and let the update accept the single directory or file (and test action should be updated the same way).

Another solution is to have two params - one for directory and another for list of yaml files.

What do you prefer?